### ISSUE TITLE
Possible richer ConnectResponse type in  '@redux-devtools/extension'

### ISSUE THREAD

**Issue by dai-shi** [reduxjs/redux-devtools#1097]
Can `ConnectResponse` in `'@redux-devtools/extension'`

https://github.com/reduxjs/redux-devtools/blob/b82de745928211cd9b7daa7a61b197ad9e11ec36/packages/redux-devtools-extension/src/index.ts#L187-L190

be compatible with or imported from

https://github.com/reduxjs/redux-devtools/blob/b82de745928211cd9b7daa7a61b197ad9e11ec36/extension/src/app/api/index.ts#L522-L536

which seems more precise?

**Comment by Methuselah96** [comment]
Yes, good catch. Ideally we would share all the types, but I'm not sure how straightforward that will be. Feel free to open a PR to update them and/or share them.
